code-challenges.test.js

```diff
@@ -1,5 +1,8 @@
 // ASSESSMENT 4: JavaScript Coding Practical Questions with Jest
 
+const { exportAllDeclaration } = require("@babel/types")
```

Comment:
Not sure if these yarg helpers (on lines 3 and 4)auto-produce by vscode gave you any problems. If you comment this out, they shouldn't interfere with your tests.